code/data-pipeline/src/data_pipeline/loader.py

```diff
@@ -67,20 +67,23 @@ def load_flusurv_rates_base(self,
     dat['inc'] = dat.weeklyrate
     dat['location'] = dat['region']
     dat['agg_level'] = np.where(dat['location'] == 'Entire Network', 'national', 'site')
-    dat = dat[(dat.age_label.isin(age_labels)) & (dat.location.isin(locations))]
```

Comment:
filters on location and season are moved to happen after we load data from 2022/23.  Previously, 2022/23 data included all locations even if a subset were requested.